docs/Build-for-Linux.md

```diff
@@ -29,7 +29,7 @@ This document assumes 'harmony' as the root directory. _JerryScript_, _libtuv_ a
 You need to install some packages to build IoT.js, as follows;
 
 ```
-sudo apt-get install gyp cmake build-essential valgrind
+sudo apt-get install gyp cmake build-essential valgrind libbluetooth-dev
```

Comment:
``libbluetooth-dev`` is always required because we don't have the feature to set which library is needed to each module.

In file ``build.config``, shared library set like below:
```
  "shared_libs": {
    "os": {
      "linux": ["m", "rt", "bluetooth"],
      "darwin": [],
      "nuttx": []
    }
  },
```

I think maybe we need to implement to set shared libraries for each module.